metricbeat/module/elasticsearch/ccr/_meta/data.json

```diff
@@ -1,39 +1,109 @@
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
     "elasticsearch": {
         "ccr": {
+            "auto_follow": {
+                "failed": {
+                    "follow_indices": {
+                        "count": 0
+                    },
+                    "remote_cluster_state_requests": {
+                        "count": 0
+                    }
+                },
+                "success": {
+                    "follow_indices": {
+                        "count": 1
+                    }
+                }
+            },
+            "bytes_read": 32768,
             "follower": {
-                "global_checkpoint": -1,
-                "index": "my_index_f",
-                "operations_written": 0,
+                "global_checkpoint": 768,
+                "index": "follower_index",
+                "max_seq_no": 896,
+                "operations": {
+                    "read": {
+                        "count": 896
+                    }
+                },
+                "operations_written": 832,
+                "settings_version": 2,
                 "shard": {
                     "number": 0
                 },
                 "time_since_last_read": {
-                    "ms": 42294
+                    "ms": 8
                 }
             },
             "leader": {
-                "index": "my_index",
-                "max_seq_no": -1
+                "global_checkpoint": 1024,
+                "index": "leader_index",
+                "max_seq_no": 1536
+            },
+            "requests": {
+                "failed": {
+                    "read": {
+                        "count": 0
+                    },
+                    "write": {
+                        "count": 0
+                    }
+                },
+                "outstanding": {
+                    "read": {
+                        "count": 8
+                    },
+                    "write": {
+                        "count": 2
+                    }
+                },
+                "successful": {
+                    "read": {
+                        "count": 32
+                    },
+                    "write": {
+                        "count": 16
+                    }
+                }
+            },
+            "total_time": {
+                "read": {
+                    "ms": 32768,
+                    "remote_exec": {
+                        "ms": 16384
+                    }
+                },
+                "write": {
+                    "ms": 16384
+                }
+            },
+            "write_buffer": {
+                "operation": {
+                    "count": 64
+                },
+                "size": {
+                    "bytes": 1536
+                }
             }
         },
         "cluster": {
-            "id": "3LbUkLkURz--FR-YO0wLNA",
-            "name": "es1"
+            "id": "",
+            "name": ""
```

Comment:
These two values being blank seems a bit odd. Is the metricset code to generate these values working properly?

Reply:
Oh, I know why this happens. It's because the mock to access '/' just returns the version of the node. I realized of the same issue in `cluster_stats` and I fixed it since there but I forgot this metricset. I'm gonna fix it now

Reply:
I mean `enrich` so actually I have to update `cluster_stats` too. Anyways, I'll probably generalize those mocks to be shared between all metricsets for some of the paths once we have all the PR's merged into the feature branch

Reply:
Ok, mock fixed! Thank you for spotting!